server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/external/flot/jquery.flot.time.js

```diff
@@ -145,8 +145,8 @@ API.txt for details.
                     case 's': c = "" + formatSubSeconds(d.getMilliseconds(), d.getMicroseconds(), decimals); break;
                     case 'y': c = leftPad(d.getFullYear() % 100); break;
                     case 'Y': c = "" + d.getFullYear(); break;
-                    case 'p': c = (isAM) ? ("" + "am") : ("" + "pm"); break;
-                    case 'P': c = (isAM) ? ("" + "AM") : ("" + "PM"); break;
+                    case 'p': c = (isAM) ? ("am") : ("pm"); break;
+                    case 'P': c = (isAM) ? ("AM") : ("PM"); break;
```

Comment:
This file should not be changed, because it's a third party library file that we've bundled. It should stay byte-for-byte equivalent to the upstream source.